### ISSUE TITLE
Two Accessibility Issues

### ISSUE THREAD

**Issue by heshamsaqqar** [labd/react-loqate#21]
the list of suggestions is not accessible with arrows, most screen readers and other assistive technologies will not understand the element.
Also, the aria-label of all the list items will have the same value (which is the search keyword) rather than the suggestions themselves (the list item's content). This will lead, for example,  to repeatedly hear the postcode numerous times, not the actual addresses to select from. See below:
![loqate1](https://user-images.githubusercontent.com/34396693/235328228-9ca9ea76-4ae3-4ca9-b20c-36e09a862a8a.png)

There is no way to customize the input to return the suggestions as a [datalist](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/datalist), that would be the ideal solution for all scenarios. 

Thank you for the great package and for the good work BTW.

**Comment by BramKaashoek** [comment]
> There is no way to customize the input to return the suggestions as a [datalist](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/datalist), that would be the ideal solution for all scenarios.

All three elements, input, list and list-item, are able to be overwritten with your own custom components.
However, I do agree that the accessibility of the default components could be improved and will do so in a future PR.
